csclient/csclient.go

```diff
@@ -79,10 +86,25 @@ func New(p Params) *Client {
 	}
 }
 
+// GetServerURL returns the charm store server URL.
+// The returned value can be overridden by setting the JujuCharmstoreEnvVar
+// environment variable.
+func ServerURL() string {
+	if url := os.Getenv(JujuCharmstoreEnvVar); url != "" {
```

Comment:
This will probably echo what @rogpeppe is saying, but it's a little odd to have library code affected by environment variables.